src/components/CountriesList.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FaRegArrowAltCircleRight } from 'react-icons/fa';
+import { getCountries } from '../redux/countriesSlice';
+
+const CountriesList = () => {
+  const dispatch = useDispatch();
+  const [searchQuery, setSearchQuery] = useState('');
+  const { countriesList, isLoading } = useSelector((store) => store.countries);
+
+  useEffect(() => {
+    dispatch(getCountries());
+  }, [dispatch]);
+
+  const handleChange = (e) => {
+    setSearchQuery(e.target.value);
+  };
+
+  const filteredCountries = countriesList.filter(({ name }) => name.common.includes(searchQuery));
+
+  if (isLoading) {
+    return <h2>Loading data...Please wait!</h2>;
+  }
+
+  return (
+    <div className="container">
+      <h1>WORLD COUNTRIES INFO</h1>
+      <form>
+        <input
+          type="search"
+          placeholder="Search by country name..."
+          value={searchQuery}
+          onChange={handleChange}
+        />
```

Comment:
- Your filter component does not work as expected, I tried searching for a country named `Barbados` in your application and got no result. Please fix this 🙏 . I suggest you consider making sure that both your search parameter and the country named being searched for are both in lower or uppercase letters, you can use the `toLowerCase()` method for this 🙏